CHANGELOG.md

```diff
@@ -4,7 +4,11 @@ Releases
 v1.0.0-dev (unreleased)
 -----------------------
 
--   No changes yet.
+-   **Breaking**: Rename `Channel` to `ClientConfig` for both the dispatcher
```

Comment:
The spacing is messed up on this one. Dedent the following lines to match up with this one (check out other multi-line entries as a guide).